git-branchless/src/opts.rs

```diff
@@ -157,6 +157,10 @@ pub enum Command {
         /// Can either be hashes, like `abc123`, or ref-specs, like `HEAD^`.
         commits: Vec<String>,
 
+        /// Also delete any branches that are abandoned as a result of this hide.
+        #[clap(short = 'D', long = "delete-branches")]
+        delete_branches: bool,
```

Comment:
[non-blocking] I had personally imagined `-b`/`--branches` as the option, but I don't feel strongly about it.

Reply:
Well, if you really don't feel strongly about it, then I would prefer it as is, but I haven't thought too much about it past the symmetry with `git branch -D`. Ultimately, it's up to you and I'll be happy to change it if you do feel strongly about it!

Reply:
Nope!